en/docs/administer/configuring-a-jdbc-user-store.md

```diff
@@ -68,103 +68,104 @@ change the `         JDBCUserStoreManager        ` section in the
         The following sample shows how to configure the internal RDBMS as
         the JDBC user store:
 
-        ``` html/xml
+        ```xml
             <UserStoreManager class="org.wso2.carbon.user.core.jdbc.JDBCUserStoreManager">
-                <Property name="TenantManager">org.wso2.carbon.user.core.tenant.JDBCTenantManager</Property>
-                <Property name="ReadOnly">false</Property>
-                <Property name="MaxUserNameListLength">100</Property>
-                <Property name="IsEmailUserName">false</Property>
-                <Property name="DomainCalculation">default</Property>
-                 <Property name="PasswordDigest">SHA-256</Property>
-                <Property name="StoreSaltedPassword">true</Property>
-                <Property name="UserNameUniqueAcrossTenants">false</Property>
-                <Property name="PasswordJavaRegEx">[\S]{5,30}$</Property>
-                <Property name="PasswordJavaScriptRegEx">[\\S]{5,30}</Property>
-                <Property name="UsernameJavaRegEx">[a-zA-Z0-9._\-|/]{3,30}$</Property>
-                <Property name="UsernameJavaScriptRegEx">[\\S]{3,30}</Property>
-                <Property name="RolenameJavaRegEx">[a-zA-Z0-9._\-|/]{3,30}$</Property>
-                <Property name="RolenameJavaScriptRegEx">[\\S]{3,30}</Property>
-                <Property name="UserRolesCacheEnabled">true</Property>
+            <Property name="TenantManager">org.wso2.carbon.user.core.tenant.JDBCTenantManager</Property>
+            <Property name="ReadOnly">false</Property>
+            <Property name="MaxUserNameListLength">100</Property>
+            <Property name="IsEmailUserName">false</Property>
+            <Property name="DomainCalculation">default</Property>
+             <Property name="PasswordDigest">SHA-256</Property>
+            <Property name="StoreSaltedPassword">true</Property>
+            <Property name="UserNameUniqueAcrossTenants">false</Property>
+            <Property name="PasswordJavaRegEx">[\S]{5,30}$</Property>
+            <Property name="PasswordJavaScriptRegEx">[\\S]{5,30}</Property>
+            <Property name="UsernameJavaRegEx">[a-zA-Z0-9._\-|/]{3,30}$</Property>
+            <Property name="UsernameJavaScriptRegEx">[\\S]{3,30}</Property>
+            <Property name="RolenameJavaRegEx">[a-zA-Z0-9._\-|/]{3,30}$</Property>
+            <Property name="RolenameJavaScriptRegEx">[\\S]{3,30}</Property>
+            <Property name="UserRolesCacheEnabled">true</Property>
```

Comment:
Shall we add the TOML configs available in https://github.com/pulasthi7/wso2-is-config/blob/master/userstores/jdbc_mysql_advance.toml.

Fix indentation.